monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -18,11 +17,11 @@ def get_random_file_suffix() -> str:
     return random_string
 
 
-def get_agent_dest_path(host: VictimHost, options: Mapping[str, Any]) -> PurePath:
+def get_agent_dest_path(host: VictimHost) -> PurePath:
```

Comment:
Might as well rename to `get_agent_dst_path` as per our guidelines of abbreviation